mindocr/data/transforms/det_transforms.py

```diff
@@ -407,6 +407,9 @@ def __call__(self, data: dict):
         if (self.limit_type in ['min', 'max']) or (self.target_size and self.keep_ratio):
             resize_w = math.ceil(w * scale_ratio)
             resize_h = math.ceil(h * scale_ratio)
+            if self.target_size:
```

Comment:
The args `target_size`  and `limit_type` are mutually excluded referring to ppocr. So we should add an assertion in __init__ to avoid setting both of them

Reply:
ok

Reply:
added in init